store.go

```diff
@@ -1786,7 +1786,7 @@ func (s *store) CreateContainer(id string, names []string, image, layer, metadat
 		options.IDMappingOptions.UIDMap = copyIDMap(cOptions.IDMappingOptions.UIDMap)
 		options.IDMappingOptions.GIDMap = copyIDMap(cOptions.IDMappingOptions.GIDMap)
 		options.LabelOpts = copyStringSlice(cOptions.LabelOpts)
-		options.Flags = maps.Clone(cOptions.Flags)
+		options.Flags = copyStringInterfaceMap(cOptions.Flags)
```

Comment:
There is already an explicit check for `nil` below, so doing that here is unnecessary.